src/main/java/io/camunda/connector/Authentication.java

```diff
@@ -0,0 +1,61 @@
+package io.camunda.connector;
+
+import io.camunda.connector.api.ConnectorInput;
+import io.camunda.connector.api.SecretStore;
+import io.camunda.connector.api.Validator;
+import java.util.Objects;
+
+public class Authentication implements ConnectorInput {
+
+  private String user;
+  private String token;
+
+  @Override
+  public void validateWith(final Validator validator) {
+    validator.require(user, "user");
+    validator.require(token, "token");
+    if (token != null && !token.startsWith("xobx")) {
+      validator.addErrorMessage("Token must start with \"xobx\"");
+    }
+  }
+
+  @Override
+  public void replaceSecrets(final SecretStore secretStore) {
+    token = secretStore.replaceSecret(token);
+  }
+
+  public String getUser() {
+    return user;
+  }
+
+  public void setUser(String user) {
+    this.user = user;
+  }
+
+  public String getToken() {
+    return token;
+  }
+
+  public void setToken(String token) {
+    this.token = token;
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(token, user);
+  }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (this == obj) return true;
```

Comment:
It's strange that auto-formatter doesn't catch such things. According to the Google Style Guide [braces are always used](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)

```suggestion
    if (this == obj) {
      return true;
    }
```